extensions/amp-photo-split-compare/0.1/validator-amp-photo-split-compare.protoascii

```diff
@@ -0,0 +1,65 @@
+#
```

Comment:
Please take a look at [contributing validator rules](https://github.com/ampproject/amphtml/blob/master/contributing/component-validator-rules.md) and update these to conform with the current standard.